static/locales/en.client.json

```diff
@@ -1948,5 +1953,14 @@
         "Have you **reviewed the code** at this URL?": "Have you **reviewed the code** at this URL?",
         "If in doubt, do not install this widget, or ask an administrator of your organization to review it for safety.": "If in doubt, do not install this widget, or ask an administrator of your organization to review it for safety.",
         "I confirm that I understand these warnings and accept the risks": "I confirm that I understand these warnings and accept the risks"
+    },
+    "AdminChecks": {
+        "Grist allows for very powerful formulas, using Python.\nWe recommend setting the environment variable\nGRIST_SANDBOX_FLAVOR to gvisor if your hardware\nsupports it (most will), to run formulas in each document\nwithin a sandbox isolated from other documents and isolated\nfrom the network.": "Grist allows for very powerful formulas, using Python.\nWe recommend setting the environment variable\nGRIST_SANDBOX_FLAVOR to gvisor if your hardware\nsupports it (most will), to run formulas in each document\nwithin a sandbox isolated from other documents and isolated\nfrom the network.",
```

Comment:
It seems this specific translation already exists and is already translated in multiple languages, but is not used anymore in the code, because it's not the same key. I feel we should handle that case so that people don't have to re-translate this?

Reply:
Allowing myself to bump this, seeing it resolved without info. We should keep in mind to update the changes in weblate after this gets merged maybe?